composer.json

```diff
@@ -22,7 +22,7 @@
         "zendframework/zend-diactoros": "^1.1",
         "zendframework/zend-expressive-router": "^1.1",
         "zendframework/zend-expressive-template": "^1.0.1",
-        "zendframework/zend-stratigility": ">=1.1.0 < 1.3.0 || >=1.3.1 <2.0.0"
+        "zendframework/zend-stratigility": ">=1.2.0 < 1.3.0 || >=1.3.1 <2.0.0"
```

Comment:
Just noticed that [the latest build](https://travis-ci.org/zendframework/zend-expressive/builds/197051503) is failing with lowest dependencies. It is because `raiseThrowables` method has beed added in Stratigility 1.3, so maybe we can just use `^1.3.3` in `composer.json` and it will resolve the issue?